### DISCUSSION TITLE
希望能增加搜索页默认打开“更多筛选”功能

### DISCUSSION THREAD

**Discussion by 233YUZI** [the1812/Bilibili-Evolved#4467] [功能建议 / Ideas]
新版首页默认折叠真的很麻烦
![image](https://github.com/the1812/Bilibili-Evolved/assets/69708373/c802433c-3d34-4530-9ee6-84be2cd7f6c5)

我自己暂时写了一个油猴脚本实现
https://greasyfork.org/zh-CN/scripts/476356
如果你们不实装就算了吧

**Comment by Geeyun-JY3** [comment]
这 b 哪儿哪儿的新版页面都爱折叠起来，也不知道有什么好藏的：#4251

**Comment by the1812** [comment]
你都做好了当然用不着再搬运一遍了
